### ISSUE TITLE
0.9.2 seed load fails with data type errors

### ISSUE THREAD

**Issue by joevandyk** [fishtown-analytics/dbt#683]
```
calendar_date,metric,reporting_category,traffic_source,customer_type,value,unit
2/4/2018,Sales, Core ,,,2,dollars
2/4/2018,Gross Margin, Non-core ,,,0.001,percent
2/4/2018,Leads,,Facebook ,,,people
2/4/2018,Sales,,,New customers,3,dollars
```
is the csv file. I'm loading into a postgresql 10 database.

```
2018-03-06 22:08:04,368: On master: drop table if exists "dbt_jvd"."plan_budget" cascade
2018-03-06 22:08:04,456: SQL status: DROP TABLE in 0.09 seconds
2018-03-06 22:08:04,457: Using postgres connection "master".
2018-03-06 22:08:04,457: On master: create table "dbt_jvd"."plan_budget" (calendar_date date, metric text, reporting_category text, traffic_source text, customer_type text, value numeric(1, 3), unit text)
2018-03-06 22:08:04,540: Postgres error: NUMERIC scale 3 must be between 0 and precision 1
LINE 1: ...t, traffic_source text, customer_type text, value numeric(1,...
                                                             ^

2018-03-06 22:08:04,541: On master: ROLLBACK
```
is the debug output from running `dbt -d seed --full-refresh`

**Comment by cmcarthur** [comment]
Thanks @joevandyk !

**Comment by cmcarthur** [comment]
Fixed by #684
